src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -76,10 +76,7 @@ function AppSettingsModal({
   }, [updateSettingsRequestStatus]);
 
   const handleFormSubmit = (values) => {
-    // If the app's enabled/disabled loadingStatus has changed, set that first.
-    if (appInfo.enabled !== values.enabled) {
-      dispatch(updateAppStatus(courseId, appInfo.id, values.enabled, true));
-    }
+    dispatch(updateAppStatus(courseId, appInfo.id, values.enabled));
```

Comment:
so now it will be dispatched even if the value is not changed right?

Reply:
Yes. The same functionality is working for legacy and LTI forms.